File: app/src/main/java/android/mb00/codehub/api/model/Readme.java
package android.mb00.codehub.api.model;

import com.google.gson.annotations.SerializedName;


public class Readme {

    @SerializedName("name")
    private String name;
    @SerializedName("size")
    private String size;
    @SerializedName("content")
    private String content;

    public String getName() {
        return name;
    }

    public String getSize() {
        return size;
    }

    public String getContent() {
        return content;
    }

}

File: app/src/main/java/mb00/android/codehub/api/model/Readme.kt
package mb00.android.codehub.api.model

import com.google.gson.annotations.SerializedName


data class Readme(
        @SerializedName("name") var name: String?,
        @SerializedName("size") var size: String?,
        @SerializedName("content") var content: String?
)
